starfish/codebook/codebook.py

```diff
@@ -546,10 +546,10 @@ def metric_decode(
         metric : str
             the sklearn metric string to pass to NearestNeighbors
 
-        See Also
-        --------
-        The available norms for this function can be found at the following link:
-        https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.linalg.norm.html
+        Notes
+        ------
+        The available norms for this function can be found `here
```

Comment:
is it worth setting up intersphinx? http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html

Reply:
I did set up intersphinx! https://github.com/spacetx/starfish/commit/1153bc42f561c4bdccd500e2847e94c89836a98a#diff-1ec61223727a10e5e4c74c68336809fcR58  But just have xarray in the mappings right now, we can add numpy too if we want

Reply:
oh cool!! yeah, if we're going to link out to NumPy docs, might as well use intersphinx to do it

Reply:
sweet I'm on it